gno.land/pkg/gnoland/types.go

```diff
@@ -20,6 +21,7 @@ func ProtoGnoAccount() std.Account {
 }
 
 type GnoGenesisState struct {
-	Balances []Balance `json:"balances"`
-	Txs      []std.Tx  `json:"txs"`
+	Balances []Balance         `json:"balances"`
+	Txs      []std.Tx          `json:"txs"`
+	Auth     auth.GenesisState `json:"auth"`
```

Comment:
This has been added as part of Manfred's recent PR to `master`

Reply:
ya,  need to merge it.